apps/andi/lib/andi_web/router.ex

```diff
@@ -32,7 +32,7 @@ defmodule AndiWeb.Router do
     pipe_through [:browser, :auth]
 
     get "/", Redirect, to: "/datasets"
-    live "/datasets", DatasetLiveView, layout: {AndiWeb.LayoutView, :root}
+    live "/datasets", DatasetLiveView, layout: {AndiWeb.LayoutView, :root}, session: {AndiWeb.Auth.TokenHandler.Plug, :current_resource, []}
```

Comment:
Is the session thing here even necessary since you are putting the userid in the session?